modules/gdscript/gdscript.h

```diff
@@ -330,6 +330,7 @@ class GDScriptInstance : public ScriptInstance {
 	Variant debug_get_member_by_index(int p_idx) const { return members[p_idx]; }
 
 	virtual void notification(int p_notification);
+	virtual void notification(int p_notification, bool p_reversed = false);
```

Comment:
Code does not make sense, why not remove the old function?

Reply:
Is it not required to be able to communicate with old extensions?

Reply:
I believe, that I have addressed this (and also https://github.com/godotengine/godot/pull/78634#discussion_r1298122129) in update https://github.com/godotengine/godot/compare/dfebd2b671e69d5142595265eded82949dafaa1e..36d8b2d8dc68fd8a3d1d83be589a12642d53506f

I'm not sure, how I could apply `ClassDB::bind_compatibility_method` of #80168 to this case, because, this `notification`-function is not registered in `ClassDB`. So this change doesn't cause any compaitbiility-breakage-message in the github-checks.